example/app.js

```diff
@@ -29,8 +43,22 @@ var App = React.createClass({
           <div className="input">
             <input type="text" value={this.state.m.format('llll')} readOnly />
           </div>
+          <div className="input">
```

Comment:
I think this should be below the calendar - since most people probably won't use the feature and it might confuse them.

Reply:
Done